dist/app.css

```diff
@@ -362,22 +368,40 @@ h6, .h6, .zeta {
   font-weight: 700; }
 
 h1, .h1, .alpha {
-  font-size: 30px; }
+  font-size: 1.96154em; }
```

Comment:
What's happened with these changes?

Reply:
A while back I had made a change so that the fonts were sized in ems, and that they get slightly smaller under a certain breakpoint. 

This time I actually regenerated the stylesheets and checked them in.